apps/e2e/cypress/e2e/back-office-applications/search.spec.js

```diff
@@ -36,13 +36,11 @@ describe('Search', () => {
 
 		it('Inspector user should be able to use search using Case Name', () => {
 			applicationsHomePage.searchFor(projectInfo.projectName);
-			searchResultsPage.verifySearchResultsCount(1);
```

Comment:
would be good to have this check working

Reply:
Yes but now changed our project name to static data...when you search, we are going to get many search results because of same name. Therefore it is not a valid any more.